resources.js

```diff
@@ -1,15 +1,74 @@
 export class Media {
-  // TODO
+constructor(title) {
+    this._title = title;
+    this._isCheckedOut = false;
+    this._ratings = [];
+  } 
+
+  get title() {
+    return this._title;
+  }
+  get isCheckedOut() {
+    return this._isCheckedOut;
+  }
+  get ratings() {
+    return this._ratings;
+  }
+  getAverageRating() {
+    let amount = 0;
+    for (let i = 0; i < this._ratings.length; i++) {
+      amount += this._ratings[i];
+    }
+    let average = amount / this._ratings.length;
+    return average; 
```

Comment:
You could `return` that value straight without storing it in `average` variable.